jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/PathResource.java

```diff
@@ -50,15 +50,17 @@ private Path checkAliasPath()
     {
         Path abs = path;
 
+        boolean isJarArchive = uri.toASCIIString().startsWith("jar:file:");
```

Comment:
@lorban I remain convinced that we should have a `JarPathResource` class to reduce these conditionals and to do other optimizations so that both Path and Jar Path resources are faster.

Reply:
1. When a path in a `jar:file:` is a directory, the URI produced will not end in a slash.
2. There is no support for symlinks in zipfs, so those tests are moot and can be skipped.

Reply:
@gregw `MountedPathResource` actually is managing only mounted JAR resources. I tried at first to make it more generic but that proved useless as the implementation progressed. So we already have the equivalent to that `JarPathResource` and I'm not against renaming it.